### PR TITLE
CF-102 Allow leaflets to use raw html

### PR DESCRIPTION
**Pull request by cgiHarland**
The text fields used for additional information for each leaflet are now rich edit fields.

Path changed in defence leaflet to point at the correct folder.

### CODE REVIEW

File: courtfinder/courts/templates/courts/leaflets/defence_leaflet.jinja [hunk 1]
```diff
@@ -37,7 +37,7 @@
                 {% include "courts/leaflets/_after_case.jinja" %}
                 {% include "courts/leaflets/_feedback.jinja" %}
                 {% include "courts/leaflets/_additional.jinja" %}
-                {% include "courts/leaflets/prosecution/_info.jinja" %}
```
Comment (robyoung) [inline courtfinder/courts/templates/courts/leaflets/defence_leaflet.jinja]: The changes in this file seem unrelated. Are they definitely correct? If so, could you reference the reason in the commit message or pull request description please?

Comment (cgiHarland) [inline courtfinder/courts/templates/courts/leaflets/defence_leaflet.jinja]: Comment amended.

Comment (zaparka) [inline courtfinder/courts/templates/courts/leaflets/defence_leaflet.jinja]: @robyoung Are you happy with the PR now ?
